presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -154,6 +154,9 @@
 
     private boolean preloadSplitsForGroupedExecution;
 
+    private boolean writingStagingFilesEnabled;
+    private int maxConcurrentFileCommits = 100;
```

Comment:
Do we want to have it as a separate properties? Why don't we reuse the `maxConcurrentFileRenames`?

Reply:
If we inject `renameExecutor` then we could reuse the same config. Let me do that.